src/sg/gov/moh/pdt-healthcert/2.0/endorsed-wrapped.json

```diff
@@ -302,60 +302,59 @@
     },
     "issuers": [
       {
-        "name": "da8347c4-ffd7-4475-9a97-6a00afc34588:string:SAMPLE ISSUER (DO NOT VERIFY)",
-        "id": "1df68f9d-6bfd-4136-a10c-072a079f97c1:string:did:ethr:0xE39479928Cc4EfFE50774488780B9f616bd4B830",
+        "name": "ba510dc9-fdf0-41ef-9d24-047c177f4e39:string:SAMPLE ISSUER (DO NOT VERIFY)",
+        "id": "f1acea0f-e952-4821-a5c1-993dc30795c6:string:did:ethr:0xE39479928Cc4EfFE50774488780B9f616bd4B830",
         "revocation": {
-          "type": "ad118832-f31a-47fb-86a4-d4f8becf710d:string:OCSP_RESPONDER",
-          "location": "e2341ef4-5659-4a6b-9100-c73afb012f3d:string:https://ocsp-responder.stg.notarise.io"
+          "type": "52d557f4-7c8c-4016-a48f-4570d278c22d:string:NONE"
```

Comment:
Have set the revocation type to `NONE` here so we don't rely on `https://ocsp-responder.stg.notarise.io`